code/modules/cargo/goodies.dm

```diff
@@ -3,7 +3,7 @@
 	access = NONE
 	group = "Goodies"
 	goody = TRUE
-
+/* // SKYRAT EDIT START - CARGO GUN
 /datum/supply_pack/goody/dumdum38
```

Comment:
Hey, what the fuck, how's the Detective supposed to buy his alternative ammo types now?

Reply:
The ammo types are available under the ammo merchant iirc

Reply:
Armament Dynamics, the cheapest company, sells the speedloaders